src/__tests__/components/Feed.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Provider } from "react-redux";
+
+import { mount } from "../../enzyme";
+import configureStore from "redux-mock-store";
+import renderer from 'react-test-renderer';
+
+import Feed from "../../components/Feed";
+
+const mockStore = configureStore([]);
```

Comment:
Why do you need to use mock of store here? Which component that are using redux?

Reply:
i think i have typed here and forgot to hit **comment**  :P